proto/umee/incentive/v1/incentive.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package umeenetwork.umee.incentive.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/incentive";
+
+// Params defines the parameters for the incentive module.
+message Params {
+  option (gogoproto.equal) = true;
+
+  // max_unbondings defines the maximum amount of concurrent unbondings an address can have.
+  uint32 max_unbondings = 1;
+
+  // unbonding_duration_long defines the unbonding duration (in seconds) of the long tier.
+  uint64 unbonding_duration_long = 2;
+
+  // unbonding_duration_middle defines the unbonding duration (in seconds) of the middle tier.
+  uint64 unbonding_duration_middle = 3;
+
+  // unbonding_duration_short defines the unbonding duration (in seconds) of the short tier.
+  uint64 unbonding_duration_short = 4;
```

Comment:
I think the values above should be immutable, yes? Let's clarify it in the comment and later assert it when we will implement the keeper.

Reply:
I see no problems with keeping it as a parameter as originally intended